### PR TITLE
handle no comments in document topic

### PR DESCRIPTION
**Pull request by brunobesson**
* use config-defined link to forums
* handle case when there is a topic for the document, but no post yet

relates to #1859

### CODE REVIEW

File: config/bruno [hunk 1]
```diff
@@ -1,7 +1,12 @@
 include config/dev
+<<<<<<< 69ea292da60cf6a4790cfa85e3478bb599d2c839
```
Comment (asaunier) [inline config/bruno]: Are you sure you want to merge conflicts in your config file? :P

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,6116 @@
+{
```
Comment (asaunier) [inline package-lock.json]: You don't need to commit this file if it is generated.

File: config/bruno (hunk 1 above)
Comment (brunobesson) [reply config/bruno]: F** another rebasing issue. I should have been more careful, I apologize.

**Comment by brunobesson** [comment]
@asaunier back to normal state... thanks for your reviews!

**Pull request merged**